R/a11y_panel.R

```diff
@@ -0,0 +1,250 @@
+          ))
+        )
+      },
+      shiny::tags$h2("Feedback and contact information"),
+      shiny::tags$p(
+        "If you need information on this website in a different format please see the parent",
+        "publications",
+        external_link(
+          href = "https://explore-education-statistics.service.gov.uk/find-statistics",
+          "on Explore education statistics"
+        ),
+        ", as detailed on the data sources page of this service.",
+        ". More details are available on that service for alternative formats of this data.",
+      ),
+      shiny::tags$p("We're always looking to improve the accessibility of this website.
+             If you find any problems not listed on this page or think we're not meeting
+             accessibility requirements, contact us:"),
+      shiny::tags$ul(tags$li(
+        shiny::tags$a(
+          href = "mailto:explore.statistics@education.gov.uk",
+          "explore.statistics@education.gov.uk"
+        )
+      )),
+      shiny::tags$h2("Enforcement procedure"),
+      shiny::tags$p(
+        "The Equality and Human Rights Commission (EHRC) is responsible for enforcing the Public ",
+        "Sector Bodies (Websites and Mobile Applications) (No. 2) Accessibility Regulations 2018",
+        "(the \"accessibility regulations\")."
+      ),
+      shiny::tags$p(
+        "If you are not happy with how we respond to your complaint, ",
+        external_link(
+          href = "https://www.equalityadvisoryservice.com/",
+          "contact the Equality Advisory and Support Service (EASS)"
+        ),
+        "."
+      ),
+      shiny::tags$h2("Technical information about this website's accessibility"),
+      shiny::tags$p(
+        "The Department for Education (DfE) is committed to making its website accessible, in ",
+        "accordance with the Public Sector Bodies (Websites and Mobile Applications) (No. 2) ",
+        "Accessibility Regulations 2018."
+      ),
+      shiny::tags$h3("Compliance status"),
+      if (all(is.na(specific_issues))) {
+        shiny::tags$p(
+          "This website is fully compliant with the ",
+          external_link(
+            href = "https://www.w3.org/TR/WCAG22/",
+            "Web Content Accessibility Guidelines version 2.2 AA standard"
+          ),
+          "."
+        )
+      } else {
+        shiny::tagList(
+          shiny::tags$p(
+            "This website is partially compliant with the ",
+            external_link(
+              href = "https://www.w3.org/TR/WCAG22/",
+              "Web Content Accessibility Guidelines version 2.2 AA standard"
+            ),
+            " due to the non-compliances listed below."
+          ),
+          shiny::tags$h3("Non accessible content"),
+          shiny::tags$p("The content listed below is non-accessible for the following reasons.
+             We will address these issues to ensure our content is accessible."),
+          shiny::tags$div(tags$ul(
```

Comment:
Minor thing, though EES does this with an ordered list, which I guess is easier to reference if anyone does want to contact us about an issue with point X?

![image](https://github.com/user-attachments/assets/ede6b135-0aa6-4700-81f6-52640d12cccd)

```suggestion
          shiny::tags$div(tags$ol(
```

Reply:
Done (before I scrolled down to see the suggested change option!)